package.nls.json

```diff
@@ -166,6 +167,14 @@
     "AWS.command.ecr.createRepository": "Create Repository...",
     "AWS.command.ecr.deleteRepository": "Delete Repository...",
     "AWS.command.ecr.deleteTag": "Delete Tag...",
+    "AWS.command.runCommandInContainer": "Run command in container",
+    "AWS.command.ecs.runCommandInContainer.warnBeforeExecute": "Command may modify the running container {0}. Are you sure?",
+    "AWS.command.ecs.runCommandInContainer.yesDoNotShowAgain": "Yes, and don't ask again",
```

Comment:
Nit: this matches `AWS.message.prompt.defaultRegionHidden.alwaysAdd`; can you make this generic and combine the strings?

Reply:
Combined to a more generic name and replaced all instances that need it.